sites/fast-website/src/public/index.html

```diff
@@ -14,20 +14,26 @@
       <div class="wrapper">
         <!-- header:start -->
         <header class="header">
-          <site-navigation>
-            <fast-anchor slot="start" appearance="lightweight" href="#">FAST</fast-anchor>
-            <fast-anchor class="header-navigation-item" appearance="lightweight" href="#">Components</fast-anchor>
-            <fast-anchor class="header-navigation-item" appearance="lightweight" href="#">Integration</fast-anchor>
-            <fast-anchor class="header-navigation-item" appearance="lightweight" href="#">Documentation</fast-anchor>
-            <fast-anchor class="header-navigation-item" appearance="lightweight" href="#">Community</fast-anchor>
-            <fast-anchor slot="end" appearance="lightweight" href="#">
+          <fast-anchor class="header-brand" appearance="lightweight" href="#">
+            <%= require("svg/icon-brand.svg") %>
+          </fast-anchor>
+          <site-navigation class="header-navigation">
+            <ul>
+              <li><fast-anchor appearance="lightweight" href="#">Components</fast-anchor></li>
+              <li><fast-anchor appearance="lightweight" href="#">Integration</fast-anchor></li>
+              <li><fast-anchor appearance="lightweight" href="#">Documentation</fast-anchor></li>
+              <li><fast-anchor appearance="lightweight" href="#">Community</fast-anchor></li>
+            </ul>
+            <fast-anchor class="header-github" slot="end" appearance="lightweight" href="#">
               Github
-              <svg width="19" height="20" viewBox="0 0 19 20" xmlns="http://www.w3.org/2000/svg">
-                <path d="M6.71154 17.0776C2.09615 18.4906 2.09615 14.7226 0.25 14.2517L6.71154 17.0776ZM13.1731 19.9036V16.2581C13.2077 15.8089 13.1482 15.3574 12.9986 14.9335C12.849 14.5096 12.6127 14.123 12.3054 13.7995C15.2038 13.4698 18.25 12.3489 18.25 7.20563C18.2498 5.89046 17.754 4.62572 16.8654 3.67319C17.2862 2.52257 17.2564 1.25074 16.7823 0.121918C16.7823 0.121918 15.6931 -0.207776 13.1731 1.51605C11.0574 0.930913 8.82722 0.930913 6.71154 1.51605C4.19154 -0.207776 3.10231 0.121918 3.10231 0.121918C2.62819 1.25074 2.59844 2.52257 3.01923 3.67319C2.12396 4.63279 1.62771 5.90895 1.63462 7.23389C1.63462 12.3394 4.68077 13.4604 7.57923 13.8278C7.27554 14.148 7.04132 14.5299 6.89182 14.9486C6.74233 15.3674 6.6809 15.8135 6.71154 16.2581V19.9036" />
-              </svg>
+              <%= require("svg/icon-github.svg") %>
             </fast-anchor>
-        </site-navigation>
+          </site-navigation>
+          <div class="header-spacer"/>
```

Comment:
Instead of a dummy `<div>`: style the container (`.header`) to have the style of the bar, push the navigation out beyond the top edge, and use padding to account for the overlap.

Reply:
dummy div removed